src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -256,46 +256,30 @@ private static String formatLocalDateTime(LocalDateTime localDateTime, String pa
 
     /**
      * Formats a date in the format dd/MM/yyyy.
-     */
-    @Deprecated
-    public static String formatDate(Date date) {
-        return formatDate(convertDateToLocalDateTime(date));
-    }
-
-    /**
-     * Formats a date in the format dd/MM/yyyy.
+     *
+     * @param localDateTime the LocalDateTime object to be formatted
+     * @return a string in the format {@code dd/MM/yyyy}
      */
     public static String formatDate(LocalDateTime localDateTime) {
         return formatLocalDateTime(localDateTime, "dd/MM/yyyy");
     }
 
     /**
      * Formats a date in the format EEE, dd MMM, yyyy. Example: Sat, 05 May, 2012
-     */
-    @Deprecated
-    public static String formatDateForSessionsForm(Date date) {
-        return formatDateForSessionsForm(convertDateToLocalDateTime(date));
-    }
-
-    /**
-     * Formats a date in the format EEE, dd MMM, yyyy. Example: Sat, 05 May, 2012
+     *
+     * @param localDateTime the LocalDateTime object to be formatted
+     * @return a string in the format {@code EEE, dd MMM, yyyy}
      */
     public static String formatDateForSessionsForm(LocalDateTime localDateTime) {
         return formatLocalDateTime(localDateTime, "EEE, dd MMM, yyyy");
     }
 
     /**
-     * Formats a date in the format dd MMM yyyy, hh:mm a. Example: 05 May 2012,
-     * 2:04 PM<br>
-     */
-    @Deprecated
-    public static String formatTime12H(Date date) {
-        return formatTime12H(convertDateToLocalDateTime(date));
-    }
-
-    /**
-     * Formats a date in the format dd MMM yyyy, hh:mm a. 12:00 PM is especially formatted as 12:00 NOON
-     * Example: 05 May 2012, 2:04 PM<br>
+     * Formats a date in the format dd MMM yyyy, hh:mm a. 12:00 PM is especially formatted as 12:00 NOON.
```

Comment:
Standardise whether or not to end with full stops. Can update the format string; it doesn't match the actual one used below.